### PR TITLE
[EDA] Update automation documentation to reflect net new org regression creation

### PR DESCRIPTION
**Pull request by jofsky**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Testing Notes
`cci flow run new_org --org release` or follow the section in documentation/automation.md

### CODE REVIEW

**Comment by dgamboni** [comment]
Hey @jofsky -- As per meeting, it sounds like we'd need to update the description, since these steps are for a net-new org and not a push upgraded org. 

We had also discussed bespoke naming for the flows to reflect 'net-new' customers ("new install") vs 'existing' customers ("push upgrade") for clarity within and across teams. (see: https://salesforce.quip.com/XybbATSNO7wr#fCMACAaTQ68)

Thanks!!!

**Comment by dgamboni** [comment]
**lurch: add

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001HsXKQA0" target="_blank">W-039609</a>

File: documentation/automation.md [hunk 1]
```diff
@@ -23,6 +23,15 @@ Regression testing the latest beta:
...
     cci org browser <orgname>
 
+Regression Testing a Net New Org:
+    git checkout master
+    git pull
+    cci org scratch release <ORG> --default
+    cci flow run install_beta
+    cci task run execute_qa_apex -o managed True -o namespaced False
+    cci task run deploy_qa_config -o unmanaged False
+    cci org browser <ORG>
```
Comment (jstvz) [inline documentation/automation.md]: You'll need to update these lines.

**Comment by jstvz** [comment]
@jofsky Please update the PR title to reflect that you're adding a new flow, not just automation.

**Comment by lifewithryan** [comment]
@jofsky @jstvz -- where do we stand with this one?

File: documentation/automation.md [hunk 2]
```diff
@@ -10,20 +10,30 @@
...
-To create an org that has been push upgraded from the latest release ("Existing Org"):
+Regression Testing EDA Beta on a Net-New Org:
+
+    git checkout master
+    git pull
+    cci org scratch release <orgname> --default
+    cci flow run net_new_org
+    cci org browser <orgname>
+
+Regression Testing EDA Beta an Existing Customer Org:
```
Comment (jstvz) [inline documentation/automation.md]:
```suggestion
##### Regression Testing EDA Beta an Existing Customer Org:
```

File: orgs/release.json [hunk 3]
```diff
@@ -1,7 +1,9 @@
 {
   "orgName": "EDA - Release Test Org",
   "edition": "Enterprise",
-  "features": ["DevelopmentWave"],
+  "features": [
+    "DevelopmentWave"
+  ],
```
Comment (jstvz) [inline orgs/release.json]:
```suggestion
  "features": ["DevelopmentWave"],
```

File: orgs/release.json [hunk 4]
```diff
@@ -10,4 +12,4 @@
       "enableChatter": true
     }
   }
-}
+}
```
Comment (jstvz) [inline orgs/release.json]:
```suggestion
}
```

File: documentation/automation.md [hunk 5]
```diff
@@ -10,20 +10,30 @@
...
-To create an org that has been push upgraded from the latest release ("Existing Org"):
+Regression Testing EDA Beta on a Net-New Org:
+
+    git checkout master
+    git pull
+    cci org scratch release <orgname> --default
+    cci flow run net_new_org
+    cci org browser <orgname>
+
+Regression Testing EDA Beta on Existing Customer Org:
```
Comment (jstvz) [inline documentation/automation.md]:
```suggestion
##### Regression Testing EDA Beta on Existing Customer Org:
```

**Pull request merged**